src/Lykke.AlgoStore.Api/Controllers/AlgoStoreUsersController.cs

```diff
@@ -50,5 +53,19 @@ public AlgoStoreUsersController(ISecurityClient securityClient)
 
             return Ok(Mapper.Map<AlgoStoreUserDataModel>(result));
         }
+
+        [HttpGet("me/instances")]
```

Comment:
I think this endpoint should be in instances controller because we get instances' information. Right now users controller is used mostly for roles management